modules/lib/default.nix

```
@@ -159,4 +159,36 @@ in
     assertion = lib.versionAtLeast ctp.getModuleRelease version;
     message = "`catppuccin/nix` requires at least version ${version} of NixOS/home-manager";
   };
+
+  # list -> list -> list
+  # this is used to make old modules to new modules
+  mkCatppuccinRenamedOpt = from: to: [
+    (lib.mkRenamedOptionModule
+      (
+        from
+        ++ [
+          "enable"
+        ]
+      )
+      [
+        "catppuccin"
+        to
+        "enable"
+      ]
+    )
+
+    (lib.mkRenamedOptionModule
+      (
+        from
+        ++ [
+          "flavor"
+        ]
+      )
+      [
+        "catppuccin"
+        to
+        "flavor"
+      ]
+    )
+  ];
```

This is something I plan on adding to the actual mkCatppuccinOpt soon ™️, but it's probably good to start doing it here and get rid of some boilerplate in our modules

(It will also need to be reformatted, sorry I am lazy)

```suggestion
  mkCatppuccinRenamedOpt = {
    from,
    to,
    accentSupport ? false
  }:
  [
    (lib.mkRenamedOptionModule
      (
        from
        ++ [
          "enable"
        ]
      )
      [
        "catppuccin"
        to
        "enable"
      ]
    )

    (lib.mkRenamedOptionModule
      (
        from
        ++ [
          "flavor"
        ]
      )
      [
        "catppuccin"
        to
        "flavor"
      ]
    )
  ]
  ++ lib.optional accentSupport (
    lib.mkRenamedOptionModule
      (
        from
        ++ [
            "accent"
        ]
      )
      [
        "catppuccin"
        to
        "accent"
      ]
    );
```